template/src/screens/DemoScreen.tsx

```diff
@@ -1,12 +1,12 @@
+import useAppDispatch from '@hooks/useAppDispatch'
+import useAppSelector from '@hooks/useAppSelector'
```

Comment:
`@hooks` should not sort before `react`

Reply:
Thanks for spotting, it was missed in config